src/sphinx/extensions/index.rst

```diff
@@ -17,7 +17,7 @@ Official Extensions
 Third Parties
 =============
 
-* `Gradle plugin <https://github.com/alphagov/gradle-gatling-plugin>`_
+* `Gradle plugin <https://github.com/lkishalmi/gradle-gatling-plugin>`_
```

Comment:
Ouch, it's all true!  :+1: for an actively supported gradle plugin.  Now how can I repay that dinner?